include/onnxruntime/core/graph/graph.h

```diff
@@ -165,25 +168,14 @@ class Node {
   @remarks The graph containing this node must be resolved, otherwise nullptr will be returned. */
   const ONNX_NAMESPACE::OpSchema* Op() const noexcept { return op_; }
 
-  /**
-  Gets the function body if applicable otherwise nullptr
-  @param try_init_func_body If not already initialized, initialize the function body
-  (This is not applicable for primitive operators.)
-  Function body can be initialized in 3 cases :
-  1. For nodes of type "Fused"
-  2. For nodes which are defined as functions in the spec (example: DynamicQuantizeLinear)
-  3. For nodes which reference a model local function. These functions are defined in the model itself and
-  do not have any schema associated with them.
-  For all other cases this will always return nullptr.
-  Nodes of type "Fused" are created during partitioning and the function body
-  initialization for such nodes also happens during node creation. Therefore,
-  initialization of function body will happen via this method only in cases 2 and 3 mentioned above.
-  */
-  Function* GetMutableFunctionBody(bool try_init_func_body = true);
+  void InstantiateFunctionBody(const logging::Logger& logger);
```

Comment:
Node has a graph pointer: graph_ . You can add a GetLogger() function to this class and avoid passing in a logger pointer.